Package.swift

```diff
@@ -154,3 +142,37 @@ let package = Package(
         ),
     ]
 )
+
+if let disableJemalloc, disableJemalloc != "false", disableJemalloc != "0" {
+    package.targets += [
+        .target(
```

Comment:
Factor out common parts of this target?

Reply:
Will file issue for it and fix next time around.